subjects/localhost/README.md

```diff
@@ -0,0 +1,78 @@
+## Localhost
```

Comment:
Sure, I'll ask him.

Actually I just reused the structure created by Hamza and Zaynab trying to make it more complete and adapting it to our specific needs in terms of language (Rust instead of C++).

Also for the last part I think you are right and I've added a small sentence after "The server" paragraph, saying that the student should create a server (since it was never explicitly stated).

About `nginx` and `siege` I don't agree... those two tools are well known in the industry and very well documented, students can google it and have all the info they need in 10 seconds, also it will be very clear to them why to use them and how to do so.

This project is a quite complex one, we can't expect people that program since less than 6 months doing it... and even so it stays quite hard, I feel at this stage the student needs to be treated as a programmer that can and must find information by himself and decide good ways to program on his own.

Reply:
Nice, Lets ask Dav about the struct then and see what he thinks for me it would be ok to have it but im not sure...
About nginx and siege, ok it makes sense so i will thrust your judgment.

Reply:
Students can google what Nginx and siege are, we don't need to explain it.
We certainly should notify students that there project will certainly be tested with siege. The subject makes it look optional, but it is required for the audit.

As long as the subject is understandable - it is okay. We should aim for some consistency, but we are yet to identify what that consistency should be. So let's release it - and restructure it later once we settle on a common structure.

Reply:
Perfect, thanks a lot for the contribution.
I've made it clear `siege` will be tested during audits.